src/statistiek_hub/resources/observation_resource.py

```diff
@@ -258,3 +255,5 @@ class Meta:
         report_skipped = True
         exclude = ("id", "created_at", "updated_at")
         import_id_fields = ("measure", "spatialdimension", "temporaldimension")
+        # Iterate over chunks of 5000 objects at once
+        chunk_size = 5000
```

Comment:
Is het nog handig om deze dynamisch te maken? Zou er een reden zijn om deze te vergroten of te verkleinen? Het voelt gek om dit te hardcoden.

Reply:
nu als param toegevoegd na de imports, dus beter te vinden.